lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -125,6 +125,10 @@ for barclamp in $deactivation_list; do
     continue
   fi
 
+  if [[ $barclamp == "tempest" ]] && [[ $applied_proposals == Usage:* ]]; then
+    continue
+  fi
```

Comment:
Before the call to `crowbar "$barclamp" list`. But actually, I have a better idea:

```shell
for barclamp in $deactivation_list; do
  if [ ! -f "/opt/dell/crowbar_framework/barclamps/$barclamp.yml" ]; then
    echo "Skipping deactivation of $barclamp barclamp."
    continue
  fi
  ...
done
```